website/docs/r/oracle_network_anchor.html.markdown

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_oracle_network_anchor"
+description: |-
+  Manages a Oracle Network Anchor.
+---
+
+# azurerm_oracle_network_anchor
+
+Manages a Oracle Network Anchor.
```

Comment:
Let's change the description to be more concise:
```suggestion
Manages an Oracle Network Anchor.
```
Reference: [reference-documentation-standards.md#descriptions](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-documentation-standards.md#descriptions)